app-modules/task/src/Filament/Resources/TaskResource/Pages/ListTasks.php

```diff
@@ -66,9 +67,19 @@ public function table(Table $table): Table
                 Filter::make('my_tasks')
                     ->label('My Tasks')
                     ->query(
-                        fn ($query) => $query->where('assigned_to', auth()->id())
-                    )
-                    ->default(),
+                        fn (Builder $query) => $query->where('assigned_to', auth()->id())
+                    ),
```

Comment:
Was there a reason we removed `->default()` off of the `my_tasks` filter here?

I believe we do still want this to be the default on page load.

Reply:
I removed it because it was causing weirdness with the my team's task filter. This is why I asked Dan about programmatically removing filters and I'm still not sure about it.

Reply:
Ah, I see. You mean if you have both selected at the same time.

Let's bring the default back for now on the `my_team` filter, and we can add a TODO and a ticket to address, making sure neither filter can be on at the same time. But for now, it's an acceptable issue with the requirement of the My Tasks page being the first loaded filter.

Reply:
Actually I got this to work after speaking to Dan.